### ISSUE TITLE
Chef-server not for debian anymore?

### ISSUE THREAD

**Issue by pierreozoux** [fnichol/knife-server#37]
Hi,

I found your plugin, and I was in love, until...I tested it and noticed that it was pointing to a deadlink... And I noticed that Opscode don't support officially debian anymore?

http://www.opscode.com/chef/install/

But I managed to install it on debian, this way : 

``` Bash
apt-get update 
apt-get upgrade

echo "deb http://ftp.debian.org/debian experimental main" >> /etc/apt/sources.list
apt-get update

apt-get -t experimental install libc6

wget https://opscode-omnibus-packages.s3.amazonaws.com/ubuntu/12.04/x86_64/chef-server_11.0.8-1.ubuntu.12.04_amd64.deb

dpkg -i chef-server_11.0.8-1.ubuntu.12.04_amd64.deb

chef-server-ctl reconfigure
```

If some people are interested, I will work on a patch for your great plugin!

**Comment by christg76** [comment]
I'm having the same problem. Just wonder have you had any problems with the experimental libc6, did you install 2.17?

**Comment by tmatilai** [comment]
You don't need to upgrade libc6, just use an Omnibus package that has been built with old enough version. The [package for Ubuntu 11.04/11.10](https://opscode-omnibus-packages.s3.amazonaws.com/ubuntu/11.04/x86_64/chef-server_11.0.8-1.ubuntu.11.04_amd64.deb) should work for Debian Wheezy. (And probably the 10.04 version for Squeeze)

**Comment by fnichol** [comment]
As it currently stands, only Ubuntu and "Enterprise Linux" (i.e. RHEL, Centos, possibly Fedora and Amazon) Omnibus packages are being built by Chef Software. @tmatilai is correct that the Ubuntu package should work on Debian, but the Omnitruck API downloader won't yield a URL to that package when run on Debian.

So as a middleground, as of 756cf01292b788274977b31af2aa8bbc26263f9d, you can use a `--url` flag to specify exactly which package you want downloaded and installed. I'm hopeful that this will help anyone use this plugin with upstream Omnibus packages on a less-than-supported OS distribution.

In other words, I guess the short answer to the above question (and sorry about the long delay) is **no**, however **it may still be possible** :smile:
